### PR TITLE
LPD-38851 - Enable filter by Custom Fields for Accounts and Account Groups

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-38851

cc @smottal

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
Cc @smottal

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java [hunk 1]
```diff
@@ -195,6 +205,8 @@ public void testGetAccountsPage() throws Exception {
 		_testGetAccountsPage(
 			Collections.singletonList(accountEntry1),
 			organization.getOrganizationId());
+
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java:208]: just remove this empty space if sf doesn't complain

**Review by smottal — APPROVED** [review]
Perfect! Just an empty line removal if sf doesn't complain.

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**